python/lsst/ts/dimm/controllers/astelco_dimm.py

```diff
@@ -560,15 +559,15 @@ async def reply_loop(self):
                         command = self.running_commands.get(cmdid)
                         if command is not None:
                             command.replies.append(reply)
-                        try:
-                            handler(command=command, cmdid=cmdid, **kwargs)
-                        except Exception:
-                            self.log.exception(
-                                f"Reply handler {handler} failed on {reply!r}"
-                            )
-                        if command.done_task.done():
-                            self.running_commands.pop(cmdid)
-                        break
+                            try:
+                                handler(command=command, cmdid=cmdid, **kwargs)
+                            except Exception:
+                                self.log.exception(
+                                    f"Reply handler {handler} failed on {reply!r}"
+                                )
+                            if command.done_task.done():
+                                self.running_commands.pop(cmdid)
+                            break
```

Comment:
Good catch. Do we want to warn if no command is found? I don't think it's necessary, and it will rarely fire, but it might provide useful diagnostics if something goes wrong.